RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/operate/rs/databases/import-export/import-data.md

You can import data into an Active-Active database [from the Cluster Manager UI](#import-data-into-a-database).
When you import data into an Active-Active database, there is a special prompt warning that the imported data will be merged into the existing database.

## Continue learning with Redis University

> See the [Import data into a database on Redis Software course](https://university.redis.io/course/jbl8tjp8niwskv) for modules.

Reviewer: ```suggestion
See the [Import data into a database on Redis Software course](https://university.redis.io/course/jbl8tjp8niwskv) to learn more.
```
